models/safe/celo/safe_celo_schema.yml

```diff
@@ -0,0 +1,137 @@
+version: 2
+
+models:
+  - name: safe_celo_singletons
+    meta:
+      blockchain: celo
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'singletons', 'celo']
+    description: "Singletons addresses used with Safes"
+    columns:
+      - &address
+        name: address
+        description: "Safe contract address"
+        tests:
+          - unique
+          - not_null
+
+  - name: safe_celo_safes
```

Comment:
can we add the unique combo of columns test?

Reply:
Hi @jeff-dude, thank you for checking this PR. Two questions

1. What should I pay attention to when building this on dunesql?
2. Not sure what you mean by "unique combo of columns test".

Reply:
@danielpartida following up here if you plan to proceed on this PR?

Reply:
Yes, 100%. 

tbh, but I have not fully understood the meaning behind the unique combo test. I was looking for documentation on the migration for DuneSQL spells but did not find something useful yet. I just have not had the time to look at the  DuneSQL Migration tag PR to have an idea of what to change.

Reply:
okay, let me help on this PR and we'll keep the scope narrow to adding celo on legacy spark.

as for the migration, please check out the readme, we put migration info at the top. hopefully that gives a good review.